python/lsst/validate/drp/photerrmodel.py

```diff
@@ -128,13 +128,20 @@ def fitPhotErrModel(mag, mag_err):
           0.039,  # gamma ('')
           24.35]  # m5 (mag)
 
-    fit_params, fit_param_covariance = curve_fit(
-        photErrModel, mag, mag_err, p0=p0)
+    try:
+        fit_params, fit_param_covariance = curve_fit(
+            photErrModel, mag, mag_err, p0=p0)
+        sigmaSys, gamma, m5 = fit_params[:3]
+    except RuntimeError as e:
+        print("fitPhotErrorModel fitting failed with")
+        print(e)
+        print("sigmaSys, gamma, m5 are being set to NaN.")
+        sigmaSys, gamma, m5 = np.nan, np.nan, np.nan
```

Comment:
Is the setting of these to NaN the only thing that lets the user know the fit failed? Where do those print statements end up?